src/a02-assertions/verifiers/handler.js

```diff
@@ -1,6 +1,6 @@
 import { sum, sumAsync } from './index.verify.js'
 import handler from '../../../verify/handler.js'
-import '../test/index.test.js'
+import(`../test/${process.env.USE_SOLUTION ? 'solution' : 'index'}.test.js`)
```

Comment:
@simoneb do you feel this approach is valid? If so, I'll implement it in the other files as well. 

Running the `verify` script directly and adding a new env var for me sounds robust enough but I'm open to suggestions :)

Reply:
if it works, why not

Reply:
I'll do something more generic to make it work also in the new docusaurus project